scripts/build-images

```diff
@@ -34,15 +34,16 @@ EOF
 if [ "${GOARCH}" != "s390x" ]; then
 xargs -n1 -t docker image pull --quiet << EOF > build/images-cilium.txt
     ${REGISTRY}/rancher/mirrored-cilium-certgen:v0.1.8
-    ${REGISTRY}/rancher/mirrored-cilium-cilium:v1.12.4
+    ${REGISTRY}/rancher/mirrored-cilium-cilium:v1.12.5
     ${REGISTRY}/rancher/mirrored-cilium-cilium-etcd-operator:v2.0.7
-    ${REGISTRY}/rancher/mirrored-cilium-clustermesh-apiserver:v1.12.4
-    ${REGISTRY}/rancher/mirrored-cilium-hubble-relay:v1.12.4
-    ${REGISTRY}/rancher/mirrored-cilium-hubble-ui:v0.9.2
-    ${REGISTRY}/rancher/mirrored-cilium-hubble-ui-backend:v0.9.2
-    ${REGISTRY}/rancher/mirrored-cilium-operator-aws:v1.12.4
-    ${REGISTRY}/rancher/mirrored-cilium-operator-azure:v1.12.4
-    ${REGISTRY}/rancher/mirrored-cilium-operator-generic:v1.12.4
+    ${REGISTRY}/rancher/mirrored-cilium-clustermesh-apiserver:v1.12.5
+    ${REGISTRY}/rancher/mirrored-cilium-hubble-relay:v1.12.5
+    ${REGISTRY}/rancher/mirrored-cilium-hubble-ui:v0.10.0
+    ${REGISTRY}/rancher/mirrored-cilium-hubble-ui-backend:v0.10.0
+    ${REGISTRY}/rancher/mirrored-cilium-operator:v1.12.5
```

Comment:
I don't believe this image is used by the chart, is it? It only uses the generic/aws/azure version?

Reply:
Thank you for the review, @brandond. I have removed it accordingly and updated the description. PTAL